pkg/pipeline/CiCdPipelineOrchestrator.go

```diff
@@ -867,10 +872,25 @@ func (impl CiCdPipelineOrchestratorImpl) CreateApp(createRequest *bean.CreateApp
 	}
 	// Rollback tx on error.
 	defer tx.Rollback()
-	app, err := impl.createAppGroup(createRequest.AppName, createRequest.UserId, createRequest.TeamId, createRequest.AppType, createRequest.Description, tx)
+	app, err := impl.createAppGroup(createRequest.AppName, createRequest.UserId, createRequest.TeamId, createRequest.AppType, tx)
 	if err != nil {
 		return nil, err
 	}
+
+	if createRequest.Description != nil && createRequest.Description.Description != "" {
+		descriptionObj := repository3.GenericNote{
+			Description:    createRequest.Description.Description,
+			IdentifierType: repository3.AppType,
+			Identifier:     app.Id,
+		}
+		note, err := impl.genericNoteService.Save(tx, &descriptionObj, createRequest.UserId)
+		if err != nil {
+			impl.logger.Errorw("error in saving description", "err", err, "descriptionObj", descriptionObj, "userId", createRequest.UserId)
+			return nil, err
+		}
+		createRequest.Description = note
+	}
```

Comment:
put this in func ``impl.storeDescription``

Reply:
done